src/prefetch.mjs

```diff
@@ -94,7 +85,7 @@ function highPriFetchStrategy(url) {
     : fetch(url, {credentials: `include`});
 }
 
-const supportedPrefetchStrategy = support(`prefetch`)
+const supportedPrefetchStrategy = typeof document !== 'undefined' && support('prefetch')
```

Comment:
To be honest, I don't think we even need the `typeof` check here since everything else already assumes/requires that we be in a browser context. Even with this, a server-side context would have to provide global shims and/or wrap the require statement in additional `typeof` checks.

Reply:
I'm okay with dropping the `typeof` check here given the above context.